chapters/classes.tex

```diff
@@ -1560,6 +1560,10 @@ \subsection{Integer Type}\label{integer-type}
 The default \lstinline!min!- and \lstinline!max!-attributes are the minimum and maximum numbers of \lstinline!IntegerType!.
 The minimal recommended number range for \lstinline!IntegerType! is from -2147483648 to +2147483647, corresponding to a two's-complement 32-bit integer implementation.
 
+\begin{nonnormative}
+The lower bound is problematic due to integer overflow; a safe alternative is -2147483647-1.
```

Comment:
I get the point, but it might not be obvious to the reader in what way the proposed difference is safe.  How about this?
```suggestion
Note that \lstinline!-2147483648! as a Modelica expression is the negation of a number above the minimal recommended number range.
To express the lower limit as a Modelica expression one may write \lstinline!-2147483647 - 1! instead.
```

Reply:
As an alternative to _more safe_ one could say:
```
To express the lower limit with all intermediate results within the minimal recommended number range, one may write \lstinline!-2147483647 - 1! instead.
```